appcues/src/main/java/com/appcues/DeepLinkHandler.kt

```diff
@@ -142,12 +146,14 @@ internal class DeepLinkHandler(scope: AppcuesScope) {
         if (!extras.getBoolean(NOTIFICATION_TEST_EXTRA)) {
             analyticsTracker.track(
                 PushOpened.eventName,
-                properties = mapOf(
+                properties = mapOf<String, Any?>(
                     "notification_id" to extras.getString(NOTIFICATION_ID_EXTRA, null),
-                    "workflow_id" to extras.getString(NOTIFICATION_WORKFLOW_ID, null),
-                    "workflow_task_id" to extras.getString(NOTIFICATION_WORKFLOW_TASK_ID, null),
+                    "push_notification_version" to extras.getInt(NOTIFICATION_VERSION_EXTRA).let { if (it == 0) null else it },
```

Comment:
I think `0` could be a valid version in some case? I'm not sure, but wondering if there is a reason we should filter that out here.

Reply:
it looks like an unpublished draft has version 0. Can we use -1 and filter that out? -1 should never be an actual value.